gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```

Comment:
May I ask you about the reason why we'd better to put `Xmx64m` here?

Reply:
This change is since gradle 5 and Gradle generated this.

This change is described here.
https://docs.gradle.org/current/userguide/upgrading_version_4.html#rel5.0:default_memory_settings

Reply:
@wreulicke Got it, thanks. I just curious about it.